NDB.Covid19/NDB.Covid19.Droid/Views/InfectionStatus/InfectionStatusActivity.cs

```diff
@@ -471,12 +471,15 @@ public async void ShowBackgroundActivityDialog()
         {
             if(await _viewModel.IsRunning())
             {
-            
+
+                string messageCombined = INFECTION_STATUS_BACKGROUND_ACTIVITY_DIALOG_MESSAGE_PART1 + "\n\n" +
```

Comment:
Is the reason for splitting these only to combine them together with `\n\n`? I am pretty sure it is possible to do it in the text file in one JSON variable, so you do not need to split these.

Reply:
It was confusing for the translation when `\n\n` was in the text file, so it was decided to separate the message in the dialog.